week-1/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,13 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+function shout() {
+  var x = ["Daniel", "Irina", "Mimi", "Rob", "Yohannes"];
+  var size = x.length;
+  for (i = 0; i < size; i++) {
```

Comment:
Great solution! Saves you typing out `console.log()` 5 times!

Reply:
Thank you!